addons/website_forum/models/forum.py

```diff
@@ -54,14 +53,20 @@ def _get_default_faq(self):
     welcome_message = fields.Html(
         'Welcome Message',
         translate=True,
-        default = """<section class="bg-info" style="height: 168px;"><div class="container">
-                        <div class="row">
-                            <div class="col-lg-12">
-                                <h1 class="text-center" style="text-align: left;">Welcome!</h1>
-                                <p class="text-muted text-center" style="text-align: left;">This community is for professionals and enthusiasts of our products and services. Share and discuss the best content and new marketing ideas, build your professional profile and become a better marketer together.</p>
-                            </div>
-                            <div class="col-lg-12">
-                                <a href="#" class="js_close_intro">Hide Intro</a>    <a class="btn btn-primary forum_register_url" href="/web/login">Register</a> </div>
+        default="""<section class="bg-info shadow">
+                        <div class="container py-5">
+                            <div class="row">
+                                <div class="col-lg-12">
+                                    <h1 class="text-center">Welcome!</h1>
+                                    <p class="text-400 text-center">
```

Comment:
We should not use text-400 inside a bg-info since if the user changes the info color to light color.... this will break... but well, I guess this is fixable by the user :/